src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Main from './components/Main';
+import NavBar from './components/NavBar';
+
+const App = () => (
+  <Router>
+    <div className="box-border text-white bg-lightpink pb-3 mt-0">
+      <NavBar />
+      <Routes>
+        <Route exact path="/" element={<Main />} />
```

Comment:
- [ ] As requested by the previous reviewer, let's add the path to the other 2 pages in the project here.
- **REQUIREMENT** The category item detail page has its unique route within the SPA.